docs/APP_widget/ver.md

```diff
@@ -6,18 +6,35 @@
 
 *请求方式: GET*
 
+鉴权方式（非必须）：[APP API 签名与鉴权](../misc/sign/APP.md)
+
 **URL参数:**
 
 | 参数名 | 类型 | 内容 | 必要性 | 备注 |
 | - | - | - | - | - |
-| appkey | str | APP 密钥 | 不必要 |  |
-| build | int | 当前 APP 版本号 | 必要 | 将会影响是否有新版本 |
-| mobi_app | str | 目标客户端类型 | 必要 | 如 `android` |
-| nt | str | 任意非空串 | 必要 | 默认为 `1` |
-| ov | str | 任意非空串 | 必要 | 默认为 `22` |
-| platform | str | 设备平台 | 必要 | 任意非空串即可, 如 `android` |
-| sn | int | 设备序列号? | 必要 | 任意数字即可, 如 `4462369` |
-| vn | str | 当前版本号 | 必要 | 任意非空串即可, 如 `6.13.0` |
+| abi | str | 设备 CPU ABI | 不必要 | 如 `arm64-v8a` |
+| appid | str | 客户端类型 | 不必要 | 如 `tv.danmaku.bili` |
+| brand | str | 设备品牌 | 不必要 | 如 `google` |
+| build | int | 当前 APP 版本号 | 必要 | 如 `8000200` |
+| channel | str | APP 分发渠道 | 必要 | 如 `master` |
+| deviceid | str | 设备唯一 ID | 不必要 | 即 XU-Prefixed BUVID，见 [device_identity](../misc/device_identity.md) |
+| env | str | APP 分发环境 | 不必要 | 默认 `prod` |
+| iv | int | 当前 APP innerVer | 不必要 | 如 `8000210` |
+| mobi_app | str | APP mobi_app | 必要 | 如 `android` |
+| model | str | 设备型号 | 不必要 | 如 `Pixel 2 XL` |
+| nt | str | - | 必要 | 默认 `1` |
+| ov | str | 设备 OS 版本 | 必要 | 安卓即 API Level，如 Android 11 为 `30` |
+| platform | str | 设备平台 | 不必要 | 如 `android` |
+| screen | str | 设备屏幕参数 | 不必要 | 如 `2712_1440@537.882_537.882`
+| sn | int | 当前 APP SN | 必要 | 如 `15088473` |
+| vn | str | 当前 APP 版本名称 | 必要 | 如 `8.0.0` |
+
+**标头参数（Headers）：**
+
+| 参数名 | 类型 | 内容 | 必要性 | 备注 |
+| - | - | - | - | - |
+| app-key | str | APPKey 名称 | 必要 | 大体等同于 [`mobi_app`](../misc/sign/APPKey.md)。特别地：64 位版本粉版客户端应为 `android64` |
+| buvid | str | BUVID | 必要 | XU-Prefixed BUVID，见 [device_identity](../misc/device_identity.md) |
```

Comment:
![image](https://github.com/user-attachments/assets/fd1c9bb9-6bfe-4d26-af5f-5f5b983b9b76)
64bit, 8.11.0

Without buvid:
![image](https://github.com/user-attachments/assets/907eee68-6a7d-48eb-bcbe-4768d594111e)
64bit, 8.1.0

Without app-key:
![image](https://github.com/user-attachments/assets/cf560311-d0d0-4b4b-a482-501dd50a7df8)
32bit, 8.11.0

Reply:
You are right. It is my problem, I have not noticed `buvid` before.
But in before documents, if a missing param or header do not lead to invalid response, it can be marked as `optional`. I think you can add these things to `Note` instead of simply `required`.

Reply:
> You are right. It is my problem, I have not noticed `buvid` before. But in before documents, if a missing param or header do not lead to invalid response, it can be marked as `optional`. I think you can add these things to `Note` instead of simply `required`.

In case of that missing any of the two headers causes invalid response (wrong ABI or version out of date), marking as `require` makes sense.

Let @SocialSisterYi or any collaborator who has write access make the final decision.